src/ui/actions_main_area.ui

```diff
@@ -282,6 +285,9 @@
            <property name="toolTip">
             <string>&lt;p&gt;Type in one or more commands you want the button to send directly to the game when this button goes from the &lt;i&gt;down&lt;/i&gt; to &lt;i&gt;up&lt;/i&gt; state.&lt;/p&gt;&lt;p&gt;To send more complex commands, that could depend on or need to modifies variables within this profile a Lua script should be entered &lt;i&gt;instead&lt;/i&gt; in the editor area below.  Anything entered here is, literally, just sent to the game server.&lt;/p&gt;&lt;p&gt;It is permissable to use both this &lt;i&gt;and&lt;/i&gt; a Lua script - this will be sent &lt;b&gt;before&lt;/b&gt; the script is run.&lt;/p&gt;</string>
            </property>
+           <property name="placeholderText">
+            <string>Text to send to the game as-is (optional)</string>
```

Comment:
Conversely - this one is only visible when the button is a push-down button - so its text only needs to compliment the *second* of the two cases in the other field.